share/solaar/io.github.pwr_solaar.solaar.metainfo.xml

```diff
@@ -5,6 +5,9 @@
 
   <name>Solaar</name>
   <summary>Linux manager for Logitech keyboards, mice, and trackpads</summary>
+  <developer id="pwr-solaar.github.io">
```

Comment:
```suggestion
  <developer id="pwr-solaar.github.io/Solaar">
```

Reply:
OK, I guess.

Reply:
Then isn't github.com/pfps a better value?

Reply:
That doesn't resolve to anything, so I don't think it is a good value, unless there are lots of examples that also use this.

Reply:
Maybe my actual Github user id 6618235?

http://caius.github.io/github_id/?github_username=pfps#pfps

But what to add to say that the number is a Github user id?

Reply:
From https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-developer: 

"The <developer/> element is designed to represent the developers or project responsible for development of the project described in the metadata.

    The element should have a id property, containing a unique ID to identify the respective developer. It is recommended to use a web URL, like gnome.org as ID to achieve a higher chance of uniqueness."

github.com/pfps is a URL so it should be fine.

Reply:
> The `URL` there means `website.tld`. Just a bad choice of word.

Grrr.   A *very* bad choice of word.

Reply:
pwr-solaar.github.io and pwr-Solaar work for me.